### ISSUE TITLE
poco -new ~/tmp/foo/bar fails when inside the Poco app directory, but not outside it

### ISSUE THREAD

**Issue by pococms** [pococms/poco#34]

**Comment by pococms** [comment]
Actually not true. It's worse.

In these examples the poco application directory is in `cd ~/pococms/poco`

```
$ rm -rf ~/foo/bar
$ cd ~/pococms/poco
$ poco -new ~/foo/bar
PocoCMS /Users/tom/pococms/poco/index.md:
 	 open /Users/tom/pococms/poco/.poco/themes/pocodocs/.poco/css/newpoquito.css: no such file or directory
```
Note it's looking for a source file in some nonexistent theme directory named `/Users/tom/pococms/poco/.poco/themes/pocodocs/.poco/css`

Going to another directory gives a completely wrong output directory:

```
$ cd ~/pococms/poco/tmp
~/pococms/poco/tmp $ poco -new ~/foo/bar
```
Look carefully! Instead of `~/foo/bar` it's just the current directory!

```
Site published to /Users/tom/pococms/poco/tmp/WWW/index.html
~/pococms/poco/tmp $ 
```

**Comment by pococms** [comment]
Fixed with commit #2706dcf
